README.md

```diff
@@ -1,3 +1,16 @@
 ## Mujoco Robot Arm
 
 Experiments to learn Mujoco and setting up traditional + RL controllers for a robotic arm environment within in. 
+
+Required Libraries: 
+- NumPy
+- Mujoco
+- MatplotLib 
```

Comment:
One simple way to help users here is to create a file named `requirements.txt` in the repo with all the needed packages and versions as dependencies. Then you can do:

```
pip install -r requirements.txt
```

As you work towards making this into a "real" Python package you can install, i.e., by doing `pip install .`, you can add these as actual requirements like this: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/